### PR TITLE
completed assesment

### PR DESCRIPTION
**Pull request by Neuroblock**
I apologize for the late push,
thank you for your awesome leadership
Frank Carbo

### CODE REVIEW

File: coding-concepts.js [hunk 1]
```diff
@@ -10,42 +10,40 @@
...
 
 // --------------------2) What will this log?
 
 var greeting = "Hello World!"
 // console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I believe it will do someting with [3] in "Hello World" or ?
+// b) Verify and explain: the output was "l" which makes me assume that [3] has to do with the 3rd letter, character, and/or index.
```
Comment (sjproctor) [inline coding-concepts.js:23]: Yup exactly, the third index of the string.

File: coding-concepts.js [hunk 2]
```diff
@@ -10,42 +10,40 @@
...
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
-
+console.log(weekendDays.toUpperCase())
 
-// --------------------5) What will this log?
+// a) Your answer: What my understanding tells me, (never seen this command), is that ".toUpperCase" could execute the command to alternate the strings "saturday" & "sunday" letter case from lower case to upper case.
+// b) Verify and explain: I received an error from the Terminal.  My first troubleshoting steps was to commnet all commands.  Another Error.  Next, was to check for a typo, maybe done by me by accidentaly changing the original text, or done Bby LEARN purposely to assess my progress.
```
Comment (sjproctor) [inline coding-concepts.js:42]: Yup this returns an error. The method `.toUpperCase` works on strings. So when called on an array you get a Type Error.

File: coding-concepts.js [hunk 3]
```diff
@@ -10,42 +10,40 @@
...
 
-var dataTypes = ["number", "string", "Boolean"]
+// --------------------5) What will this log? I received an error from the Terminal.
+// var dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
 // a) Your answer:
-// b) Verify and explain:
+// b) Verify and explain: ypeof??? AN ERROR?
```
Comment (sjproctor) [inline coding-concepts.js:49]: `typeof` is an operator that return the name of the data type. It is useful for troubleshooting code.

File: interview-questions.md [hunk 4]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: True or False
 
-  Researched answer:
+  Researched answer: freecodecamp.org/
```
Comment (sjproctor) [inline interview-questions.md:20]: In the future, I would like you to research and then paraphrase your researched answer. It is a learning technique. :)

File: interview-questions.md [hunk 5]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
-  Researched answer:
+  Researched answer:  "Git is a software that tracks changes made to any set of files. Wikipedia.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: True or False
```
Comment (sjproctor) [inline interview-questions.md:18]: Well, yes Boolean values are true and false. But I am looking for what will get us a Boolean value. We know that relational, logical, and equality operators return Boolean values.

File: interview-questions.md [hunk 6]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: Proper pair programming techniques include, communication platforms, Software that tracks changes in a repository, and proper uploading through branches. Branches allows for Main code to be undisturbed.
 
-  Researched answer:
+  Researched answer: "Pair programming is an agile software development technique in which two programmers work together at one workstation. One, the driver, writes code while the other, the observer or navigator, reviews each line of code as it is typed in. The two programmers switch roles frequently"
```
Comment (sjproctor) [inline interview-questions.md:45]: Nice!

File: interview-questions.md [hunk 7]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 6. What is something we did in class this week you found helpful?  
 
-  Your answer:
+  Your answer: Working with others.
```
Comment (sjproctor) [inline interview-questions.md:51]: Yay great!

**Review by sjproctor — APPROVED** [review]
Looks good!

**Comment by sjproctor** [comment]
Good work Frank! You showed a good understanding of the material. Some feedback is to make sure you are answering the interview questions thoroughly. Take the time to research the topics and report your findings. It is an excellent way to learn.

File: coding-concepts.js (hunk 3 above)
Comment (Neuroblock) [reply coding-concepts.js:49]: typeof = data type

File: interview-questions.md (hunk 4 above)
Comment (Neuroblock) [reply interview-questions.md:20]: oh ok tracking

**Pull request merged**